media-proxy/include/session-base.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2024 Intel Corporation
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef __SESSION_BASE_H
+#define __SESSION_BASE_H
+
+#include <limits.h>
+#include <errno.h>
+#include <stdlib.h>
+#include <atomic>
+#include <thread>
+#include <stdexcept>
+
+using namespace std; // for atomic_bool
+
+#include <mcm_dp.h>
+
+#include "shm_memif.h" /* share memory */
+
+#include "utils.h"
+
+class session
```

Comment:
Let's capitalize class names

Reply:
Do you mean CamelCase or Camel_Snake ?

Reply:
I don't want to change snake_case, but personally I'm used to class names starting with capital letters, so for classes it would be Snake_case_started_with_capital_letter (however it is called) ^^

Reply:
I would suggest JustForClassNames, to be consistent with gRPC/TCP classes

Reply:
done